src/lib/djerba/render/database.py

```diff
@@ -0,0 +1,75 @@
+"""Interface with a CouchDB instance for JSON report documents"""
+
+import logging 
+import configparser 
+import requests
+import json
+import os
+
+import djerba.util.constants as constants
+import djerba.util.ini_fields as ini
+from datetime import datetime
+from djerba.util.logger import logger
+
+class Database(logger):
+    """Class to communicate with CouchDB via the API, eg. using HTTP GET/POST statements"""
+
+    def __init__(self, log_level=logging.WARNING, log_path=None):
+        self.logger = self.get_logger(log_level, __name__, log_path)
+        self.logger.info("Initializing Djerba database object")
+    
+    def Merge(self,dict1,dict2):
+        combined = {**dict1, **dict2}
+        return combined
+
+    """ Upload json to couchdb"""
+    def UploadFile(self,json_path):
```

Comment:
Fix case, and put spaces between function arguments for readability:
`UploadFile(self,json_path)` -> `upload_file(self, json_path)`